flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/functions/MathFunctions.scala

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.flink.table.functions.utils
+package org.apache.flink.table.runtime.functions
```

Comment:
I suggest `MathFunctions`->`ScalarFunctions`. so that we can add all scalar functions in one file. and in the `...runtime.functions` package we have three file `ScalarFunctions`,`TableFunctions` and `AggFunctions`.Which in FLINK-6810 plan to do. What do you think?